cla_public/templates/macros/form.html

```diff
@@ -13,10 +13,11 @@
     - attrs <object> (default: None)
         Custom HTML attributes for fieldset element
 #}
-{% macro fieldset(field=None, class_='govuk-fieldset', legend=None, field_attrs={}) %}
+{% macro fieldset(field=None, class_='govuk-fieldset ', legend=None, field_attrs={}) %}
```

Comment:
None blocking: Is the space at the end of the class attribute intentional

Reply:
Yes it is